db/event_manager.go

```diff
@@ -79,7 +106,13 @@ func (em *EventManager) ProcessEvent(event Event) {
 			defer wg.Done()
 			//TODO: Currently we're not tracking success/fail from event handlers.  When this
 			// is needed, could pass a channel to HandleEvent for tracking results
-			handler.HandleEvent(event)
+			if handler.HandleEvent(event) {
+				em.IncrementEventsProcessedSuccess()
+			} else {
+				em.IncrementEventsProcessedFail()
+			}
+			base.Tracef(base.KeyAll, "Webhook event processed %s", event.String())
```

Comment:
event.String() is reasonably expensive, and is getting executed here regardless of whether trace logging is enabled.  Changing to:
base.Tracef(base.KeyAll, "Webhook event processed %s", event)
will have the same result, but only execute event.String() when the format is actually applied.